src/pykeen/cli.py

```diff
@@ -202,7 +202,7 @@ def evaluators(tablefmt: str):
 
 
 def _help_evaluators(tablefmt):
-    lines = sorted(_get_lines(evaluators_dict, tablefmt, 'evaluation'))
+    lines = sorted(_get_lines(evaluator_resolver.lookup_dict, tablefmt, 'evaluation'))
```

Comment:
The `_help_{evaluators,stoppers,...}` look quite similar, so maybe could be even further merged.

No blocker, since they themselves are quite small.

Reply:
True, but they are used in two places so saving them as thin wrappers seems to be the best way to reduce code